utilities/transactions/pessimistic_transaction_db.cc

```diff
@@ -257,9 +257,9 @@ void TransactionDB::PrepareWrap(
   for (size_t i = 0; i < column_families->size(); i++) {
     ColumnFamilyOptions* cf_options = &(*column_families)[i].options;
 
-    if (cf_options->max_write_buffer_number_to_maintain == 0) {
+    if (cf_options->max_write_buffer_size_to_maintain == 0) {
       // Setting to -1 will set the History size to max_write_buffer_number.
-      cf_options->max_write_buffer_number_to_maintain = -1;
+      cf_options->max_write_buffer_size_to_maintain = -1;
```

Comment:
What does -1 mean here? Set to max_write_buffer_number*write_buffer_size?

Reply:
Yes, I will need to update the documentation across the board

Reply:
Has this been resolved?

Reply:
Yes